install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -65,6 +65,7 @@ spec:
 {{- if not $.Values.global.configValidation }}
           - --enable-validation=false
 {{- end }}
+          - --enable-reconcileWebhookConfiguration={{ .Values.enableReconcileWebhookConfiguration }}
```

Comment:
Can you set a default value here to avoid invalid args by adding | default <true_or_false>?

Reply:
The default value has been set in install/kubernetes/helm/istio/charts/galley/values.yaml. Moreover, if changing the line into " - --enable-reconcileWebhookConfiguration={{ .Values.enableReconcileWebhookConfiguration | default true }}", the helm set command does not set the value of enable-reconcileWebhookConfiguration based on the value in the helm set command, e.g., the command "helm template     --name=istio     --namespace=istio-system     --set galley.enableReconcileWebhookConfiguration=false     install/kubernetes/helm/istio > istio-test.yaml" will not set enable-reconcileWebhookConfiguration as false.